README.md

```diff
@@ -28,6 +28,14 @@ Alternatively, you can find the package available on
 [MELPA](http://melpa.org/#/editorconfig) and [MELPA Stable](http://stable.melpa.org/#/editorconfig)
 ([The Marmalade package](http://marmalade-repo.org/packages/editorconfig) is deprecated).
 
+Or if you use [**use-package**](https://www.emacswiki.org/emacs/UsePackage):
+
+```emacs-list
+(use-package editorconfig
+  :ensure t
+  :init
+  (add-hook 'prog-mode-hook 'editorconfig-mode))
```

Comment:
@xuhdev and @10sr , I've made this commit https://github.com/editorconfig/editorconfig-emacs/pull/103/commits/6f55f00fc0c8920c5c7b443b09f74e83b7d6ba8c to make some changes.

Please, tell me if you have any other suggestion.

Thanks!